DEVELOP.md

```diff
@@ -4,7 +4,7 @@
 
 The following dependencies are required for local development:
 
-- [Go](https://golang.org/dl/) >= 1.19.3
+- [Go](https://golang.org/dl/) >= 1.20.3
```

Comment:
```suggestion
- [Go](https://golang.org/dl/) >= 1.20.4
```

Reply:
since golang 1.19 is still *supported*, I won't change this for now. The *recommended* version is 1.20 though, which will be in the release notes.

Reply:
That’s totally fine but you should make them consistent. You use go 1.20.4 everywhere except this specific one. You shouldn’t do that.

make them consistent was my review feedback.